src/loaders/xml/mod.rs

```diff
@@ -0,0 +1,39 @@
+mod client_info;
+
+use serde::Deserialize;
+use serde_xml_rs::de::Deserializer;
+use xml::reader::{EventReader, ParserConfig};
+
+pub use self::client_info::{ClientInfo, Connection};
+use super::GameFileLoader;
+#[cfg(feature = "debug")]
+use crate::debug::*;
+
+#[derive(Default)]
+pub struct XMLFileLoader;
+
+impl XMLFileLoader {
+    pub fn load_client_info(&self, game_file_loader: &mut GameFileLoader) -> ClientInfo {
+        let clientinfo = game_file_loader.get("data\\clientinfo.xml");
+        let sclientinfo = game_file_loader.get("data\\sclientinfo.xml");
+
+        let src = match (sclientinfo, clientinfo) {
+            (Ok(data), _) => String::from_utf8(data).unwrap(),
+            (_, Ok(data)) => String::from_utf8(data).unwrap(),
+            _ => panic!("FOO"),
+        };
+
+        // TODO: Make it work with euc-kr, since it panics
+        // with error: "Unsupported encoding: euc-kr"
+        let replaced_src = src.replace("euc-kr", "utf8");
+
+        let config = ParserConfig::new().trim_whitespace(true);
+        let event_reader = EventReader::new_with_config(replaced_src.as_bytes(), config);
+        let client_info = ClientInfo::deserialize(&mut Deserializer::new(event_reader)).unwrap();
+
+        #[cfg(feature = "debug")]
+        print_debug!("reading client info:\n{}{:#?}{}", MAGENTA, client_info, NONE);
```

Comment:
I would like to remove this as it produces too many lines in the logs and is generally a bit inconsistent with the rest of the logging. My suggestion would be to add a
```rust
#[cfg(feature = "debug")]
let timer = Timer::new("read clientinfo");
```
in the beginning and
```rust
#[cfg(feature = "debug")]
timer.stop();
```
at the end.